pkg/server/api/bundle/v1/service_test.go

```diff
@@ -503,53 +503,14 @@ func TestAppendBundle(t *testing.T) {
 			expectLogs: []spiretest.LogEntry{
 				{
 					Level:   logrus.ErrorLevel,
-					Message: "Invalid request: failed to convert bundle",
+					Message: "Invalid argument: failed to convert bundle",
 					Data: logrus.Fields{
 						telemetry.TrustDomainID: serverTrustDomain.String(),
 						logrus.ErrorKey:         "unable to parse JWT authority: missing key ID",
 					},
 				},
 			},
 		},
-		{
-			name: "datasource fails",
```

Comment:
We still call the datastore to append the bundle, so I think that we should keep the datastore failure case. It needs to be updated with the "failed to append bundle" error instead of "failed to fetch server bundle".

Reply:
Right! good catch!